public/app/features/logs/components/LogDetailsRow.tsx

```diff
@@ -111,6 +120,16 @@ const getStyles = memoizeOne((theme: GrafanaTheme2, activeButton: boolean) => {
       label: wrapLine;
       white-space: pre-wrap;
     `,
+    parsedValue: css`
+      display: inline;
+    `,
+    longLineScroll: css`
+      height: ${theme.spacing(theme.components.height.sm * 2.4)};
```

Comment:
This is a magic number, but I figured showing 2ish lines of text would help the user see that there is more content that will be available on scroll.

Reply:
Setting a fixed height to the log line, and adding a scroll, seems like a significant UX change to me. Is this really required for this PR?
Looping in @grafana/observability-logs for more eyes on this.

Reply:
Just saw this is inside the LogDetails component, so probably not as impactful as I first thought.

Reply:
@matyax To be clear here, what this PR does is show the entire log line if there's a data link attached to that field. Previously, the entire log line field would always be hidden. This is the entire point to the PR

The UI change is an attempt to deal with the scenario where the log line is very long and takes up significant UI space. This is repeating the log line that is shown non-expanded, so we were looking for ways to display the context the data link has in a way that had less redundancy. You can see what this looks like without the edit here https://github.com/grafana/grafana/pull/62926/files#r1116727135 

The request was to truncate it, but I believe giving users the ability to scroll through their log line is a better approach. Truncating it would have less of a UI impact though. Hopefully this helps explain what is going on.

Reply:
Thanks for the explanation, it's clearer now.

Reply:
Would it make sense (a be simpler) to truncate the line instead of having a scroll? The line is visible above the list of fields anyway, we just need a way to show the link for the line. I'm not sure how useful at this stage would be to repeat the entire message especially if formatting (JSON) needs to be used.